### PR TITLE
Logo block and about page updates

### PR DESCRIPTION
**Pull request by PatCapon39**
@GarethPrice-Aus  @neoformit  based on email discussions these changes:

- Update site footer to have NCRIS, Bioplatforms and BioCommons (infrastructure tagline) logos
- Move all other logos to the about page
- Reformat about page to delineate types of partners (operational, computational, funding) and provide colour logos
- Remove 'the' before BioCommons
- Add about to site links
- Add partner acknowledgement statement to site footer (to cover the removal of their logos)
- Condense and shift the usegalaxy.* information

I've copied Cam into the email chain :)

### CODE REVIEW

File: webapp/home/templates/home/snippets/footer.html [hunk 1]
```diff
@@ -11,50 +11,14 @@
...
-
-    <div class="text-center second">
-      <a href="https://www.uq.edu.au" target="_blank">
-        <img src="{% static 'logos/footer/uq-grayscale.png' %}" />
-      </a>
-
-      <a href="https://ardc.edu.au" target="_blank">
-        <img src="{% static 'logos/footer/ardc-grayscale.png' %}" />
+      <a href="https://bioplatforms.com/" target="_blank">
+        <img src="{% static 'BIO-RGB_Full-POS.png' %}" />
```
Comment (neoformit) [inline webapp/home/templates/home/snippets/footer.html]: ~Image path is missing `/logos/footer/`~ Fixed

File: webapp/home/templates/home/snippets/footer.html [hunk 2]
```diff
@@ -11,50 +11,14 @@
...
-      </a>
-
-      <a href="https://www.dese.gov.au/ncris" target="_blank">
-        <img src="{% static 'logos/footer/ncris-grayscale.png' %}" />
-      </a>
-
-      <a href="https://www.qld.gov.au" target="_blank">
-        <img src="{% static 'logos/footer/qld-gov.png' %}" />
+      <a href="https://www.biocommons.org.au/" target="_blank">
+        <img src="{% static 'Australian-Biocommons-Logo-Tagline-Infrastructure-RGB-no-padding.png' %}" />
```
Comment (neoformit) [inline webapp/home/templates/home/snippets/footer.html]: ~Image path is missing `/logos/footer/`~ Fixed

File: webapp/home/templates/home/snippets/footer.html [hunk 3]
```diff
@@ -78,14 +42,15 @@ <h6>SITE LINKS</h6>
           <div class="col-md-6">
             <small>
               <a href="/request">Support</a> <br>
+              <a href="/about">About</a>a> <br>
```
Comment (neoformit) [inline webapp/home/templates/home/snippets/footer.html:45]: Typo in closing tag

File: webapp/home/templates/home/snippets/footer.html (hunk 2 above)
Comment (neoformit) [reply webapp/home/templates/home/snippets/footer.html]: Does this extra text really need to be squeezed into the Biocommons logo? If it's important, can we put this text somewhere else in the footer where it is more legible, and doesn't look like a fourth logo?

image

File: webapp/home/templates/home/snippets/footer.html [hunk 4]
```diff
@@ -100,6 +65,7 @@ <h6>SITE LINKS</h6>
             <h6>ACKNOWLEDGEMENTS</h6>
             <p class="tight">
               <small>
+                Galaxy Australia acknowledges operational and computational  support from its <a href="/about#operational-partners"> partner organisations </a>. <br>
```
Comment (neoformit) [inline webapp/home/templates/home/snippets/footer.html:68]: Mind the space in `organisations </a>.`

File: webapp/home/templates/home/snippets/about/about.html [hunk 5]
```diff
@@ -36,43 +40,99 @@ <h3 id="acknowledgement-statement">
...
+        <img src="{% static 'logos/footer/nci_logo.png' %}" />
+    </a>
+    <a href="https://pawsey.org.au" target="_blank">
+        <img src="{% static 'logos/footer/pawsey-grayscalelogo-beige.png' %}" />
+    </a>
+     <a href="https://azure.microsoft.com/en-au" target="_blank">
+        <img src="{% static 'logos/footer/Microsoft_Azure_Logo.svg.png' %}" />
+      </a>
+    </div>
+  </div>
```
Comment (neoformit) [inline webapp/home/templates/home/snippets/about/about.html:114]: These look good after fixing a couple of typos in file names

image

**Review by neoformit — CHANGES_REQUESTED** [review]

File: webapp/home/templates/home/snippets/footer.html (hunk 3 above)
Comment (PatCapon39) [reply webapp/home/templates/home/snippets/footer.html]: Fixed, thanks! That damn autocompletion of tags gets me so often!

File: webapp/home/templates/home/snippets/footer.html (hunk 4 above)
Comment (PatCapon39) [reply webapp/home/templates/home/snippets/footer.html]: Good catch, thanks. Fixed now

File: webapp/home/templates/home/snippets/about/about.html (hunk 5 above)
Comment (PatCapon39) [reply webapp/home/templates/home/snippets/about/about.html:114]: Nice! Thanks for sorting those missing file paths out

File: webapp/home/templates/home/snippets/footer.html (hunk 2 above)
Comment (PatCapon39) [reply webapp/home/templates/home/snippets/footer.html]: Hmm I will raise this one, you're right it is rather small and hard to read in this position

Comment (PatCapon39) [reply webapp/home/templates/home/snippets/footer.html]: Ok I've reverted to the regular BioCommons logo and moved the text into the footer. I removed the header of 'acknowledgements' as they're really 3 distinct statements, and the header takes up space that we need to separate the statements.

@neoformit do the <br> tags that I put leave a white line between the sentence? Or do I need to put a <p></p> in to force a white line between each line of text?

**Comment by PatCapon39** [comment]
Added an edit to remove CloudStor from the data privacy page as discussed with @GarethPrice-Aus by email

File: webapp/home/templates/home/snippets/footer.html (hunk 2 above)
Comment (PatCapon39) [reply webapp/home/templates/home/snippets/footer.html]: Ha, I see my comment actually formatted the code and answered my own question! Updated the footer code so I think it's good to go now :)

Comment (neoformit) [reply webapp/home/templates/home/snippets/footer.html]: Haha, yes! `<br>` is a line break, `<p>` is a paragraph. Usually a blank line between text blocks means that it's a new paragraph, so if that's the case then use `<p>` instead.

Comment (neoformit) [reply webapp/home/templates/home/snippets/footer.html]: This is ok:

image

But I have shuffled around the columns and spacing and I think this looks a bit neater? What do you think? Does the "Acknowledgements" title still make sense? It just looks out-of-alignment if one section doesn't have a title.

image

Comment (PatCapon39) [reply webapp/home/templates/home/snippets/footer.html]: Agreed it does look a bit odd with one section without a title, so let's leave the title there. I like your spacing shuffle, thanks for that. I'll put in more changes to fix the paragraph code, and reinstate the title, can you then please make the alignment fix?

Are the words 'partner organisations' in the second acknowledgement hyperlinked to /about#operational-partners? That's what I've tried to achieve but it's not coloured like a hyperlink - maybe that's intentional in the footer?

Comment (neoformit) [reply webapp/home/templates/home/snippets/footer.html]: Cool, I've merged our changes. I think it looks decent now:

image

Comment (PatCapon39) [reply webapp/home/templates/home/snippets/footer.html]: Agreed - I'll email Gareth and Christina to confirm they're happy with it

**Pull request merged**